### DISCUSSION TITLE
Waterleak sensor

### DISCUSSION THREAD

**Discussion by Hjorth-Jensen** [andrei-tatar/node-red-contrib-smartnora#131] [Q&A] [answered]
Hi
I'm a little beginner. I can not get sensor waterleak to work. Is there anyone who can post a small example. So I can see the setup on the payload that I need to send.

Thank you.

**Comment by andrei-tatar** [comment, marked as answer]
Hi,

The payload should have a state property with values `leak`/`no leak`. 
```
msg = {
  payload: {
    state: 'leak',
  },
};
```

Example flow:

```
[{"id":"58757c8d3b6af31f","type":"noraf-sensor","z":"b9a14f5917f73660","devicename":"Sensor","roomhint":"","name":"","temperature":false,"unit":"C","humidity":false,"passthru":false,"nora":"f1e538c1.0315f8","topic":"","filter":false,"sensorSupport":true,"sensorType":"WaterLeak","sensorStates":["leak","no leak"],"sensorNumeric":true,"openCloseSupport":false,"openCloseDiscrete":false,"onOffSupport":false,"x":970,"y":360,"wires":[]},{"id":"cc1e62ada280463d","type":"inject","z":"b9a14f5917f73660","name":"leak","props":[{"p":"payload.state","v":"leak","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","x":730,"y":320,"wires":[["58757c8d3b6af31f"]]},{"id":"9103989a5f07b441","type":"inject","z":"b9a14f5917f73660","name":"no leak","props":[{"p":"payload.state","v":"no leak","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","x":730,"y":380,"wires":[["58757c8d3b6af31f"]]},{"id":"f1e538c1.0315f8","type":"noraf-config","name":"NORA","group":"seceani","twofactor":"off","twofactorpin":"","localexecution":true,"structure":"Seceani","storeStateInContext":true}]
```

**Comment by Hjorth-Jensen** [comment]
Thank you, Andrei
And thanks for a super node-red plugin.

**Comment by Hjorth-Jensen** [comment]
And just one more question ..
I do not think I get any alarm if there is waterleak. Where will I get it?
I have google nest hub and Iphone with google home. There is no alarm, anywhere.

**Reply by andrei-tatar** [reply]
Pushing notifications to Google for sensors is not yet supported. It's something on the list that I'm working on.
https://developers.google.com/assistant/smarthome/traits/sensorstate#device-notifications

**Comment by andrei-tatar** [comment]
@Hjorth-Jensen Version `1.7.0` now supports voice notifications when sensor state changes. All you need to do is check the notification box in node-red for the states you wish to be notified and enable "Voice notifications" for that device in Google Home (jus click it and a new option will be visible).
![image](https://user-images.githubusercontent.com/6473183/145713975-410514ca-8dc5-431f-bd5a-85061feae546.png)

**Comment by Hjorth-Jensen** [comment]
Thanks 
You work fast
